doc/source/index.rst

```diff
@@ -21,9 +18,201 @@ Using and Extending Bandit
    formatters/index
    faq
 
+Contributing
+============
+
+* `Source code`_
+* `Issue tracker`_
+
+.. _`Source code`: https://github.com/PyCQA/bandit
+.. _`Issue tracker`: https://github.com/PyCQA/bandit/issues
+
 Indices and tables
 ==================
 
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
+
+License
+=======
```

Comment:
I don't believe this is what they wanted and even if it is, I'm strongly against this solution